activate.ps1

```diff
@@ -1,5 +1,5 @@
 #
-# This file must be used by invoking ". .\activate.ps1" from the command line.
+# This file must be used by invoking ".\activate.ps1" from the command line.
```

Comment:
Why? `. ./activate.ps1` is what works for me.

Reply:
Was causing an issue in my shell, however a restart seems to have resolved it. Closing.